impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js

```diff
@@ -60,6 +61,15 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_TemplatedMixin", "dij
     
     this.callbacks = [lang.hitch(this, this.save), lang.hitch(this, this.cancel)];
 
+    this.own([
+      a11yUtil.makeAccessibleActionButton(this.picklistRemoveAll),
+      a11yUtil.makeAccessibleActionButton(this.picklistAddAll),
+      a11yUtil.makeAccessibleActionButton(this.picklistRemoveSelected),
+      a11yUtil.makeAccessibleActionButton(this.picklistAddSelected),
+      a11yUtil.makeAccessibleActionButton(this.typePicklistCombinationTypeLinksIncludeLink),
+      a11yUtil.makeAccessibleActionButton(this.typePicklistCombinationTypeLinksExcludeLink)
+    ]);
```

Comment:
The `this.own( ... )` method receives _multiple, separate_ arguments, one for each "destroyable" object, and not a single array argument with many such objects.